src/type-resolution/callables.ts

```diff
@@ -97,18 +98,19 @@ export function resolveInstanceMethodCallExpression(
 
 function inferMethodCall(
   typeSystem: reflect.TypeSystem,
-  call: ObjectLiteral
+  call: ObjectLiteral,
+  logicalId?: string
 ): MethodCall {
   const candidateFQN = assertFullyQualifiedStaticMethodCall(call);
   const candidateClass = typeSystem.findClass(candidateFQN);
-  const methods = enumLikeClassMethods(candidateClass);
+  const methods = staticNonVoidMethods(candidateClass);
```

Comment:
Because methods like `Bucket.fromBucketName()` return an `IBucket` and thus get filtered out in `enumLikeClassMethods()`.